### ISSUE TITLE
Accessibility enhancement with JAWS screen reader

### ISSUE THREAD

**Issue by Bohdan-SUP** [AdguardTeam/AdguardForAndroid#1120]
User 1221653 reported:

> As a JAWS screen reader user, I am **unable to enter my license key to activate** AdGuard due to a pop up dialogue box which blocks the edit field. Would it be possible for the devs to download a free demo version of JAWS from the link below AND kindly address this issue.

http://www.freedomscientific.com/Downloads

**Comment by TPS** [comment]
Does #957 help w/ this @ all?

**Comment by ameshkov** [comment]
@Revertron could you please check it?

**Comment by Revertron** [comment]
And where is JAWS reader for Android?

**Comment by ameshkov** [comment]
@Bohdan-SUP are you sure this is about Android?

**Comment by Bohdan-SUP** [comment]
There was a confusion indeed, updated https://github.com/AdguardTeam/AdguardForWindows/issues/352
